scipy/special/meson.build

```diff
@@ -0,0 +1,464 @@
+    'orthogonal_eval.pxd',
+    'sf_error.pxd',
+    'sph_harm.pxd', 
+    '_cython_special.pxi',
+    '_cython_special_custom.pxi',
+    '_ufuncs_extra_code.pxi',
+    '_ufuncs_extra_code_common.pxi',
+  ],
+  command : ['cp', '@INPUT@', '-t', '@OUTDIR@'],
+)
+
+py3.extension_module('_ufuncs',
+  [ufuncs_sources, _generate_cy[0], _ufuncs_pxi_pxd, _dummy_init],
+  c_args : [numpy_nodepr_api, use_math_defines],
+  include_directories: [inc_np, '../_lib', '../_build_utils/src'],
+  dependencies : [py3_dep, lapack, npymath_lib],
+  link_with: [
+    amos_lib,
+    cdflib_lib,
+    cephes_lib,
+    mach_lib,
+    specfun_lib,
+  ],
+  install : true,
+  subdir : 'scipy/special'
+)
+
+_ufuncs_cxx_cy = custom_target('_ufuncs_cxx_cy',
+  output : '_ufuncs_cxx.cpp',
+  input : _generate_cy[2],
+  command : [cython, '--cplus', '-3', '--fast-fail', '@INPUT@']
+)
+
+py3.extension_module('_ufuncs_cxx',
+  [ufuncs_cxx_sources, _ufuncs_cxx_cy, _ufuncs_pxi_pxd, _dummy_init],
+  cpp_args : [numpy_nodepr_api, use_math_defines],
+  include_directories: [inc_np, '../_lib', '../_build_utils/src'],
+  dependencies : [py3_dep, npymath_lib],
+  install : true,
+  subdir : 'scipy/special'
+)
+
+cython_special_pxd = declare_dependency(sources: _generate_cy)
+special_pxd = declare_dependency(sources: _special_pxd)
+_ellip_harm_pyx = custom_target('_ellip_harm_pyx',
+  output : '_ellip_harm_2.pyx',
+  input : '_ellip_harm_2.pyx',
+  command : ['cp', '@INPUT@', '@OUTDIR@'],
+)
+
+py3.extension_module('_ellip_harm_2',
+  [_ellip_harm_pyx, 'sf_error.c'],
+  c_args : numpy_nodepr_api,
+  include_directories: [inc_np, '../_lib', '../_build_utils/src'],
+  dependencies : [py3_dep, lapack, special_pxd],
+  install : true,
+  subdir : 'scipy/special')
+
+py3.extension_module('cython_special',
+  [_generate_cy[6],  # cython_special.pyx
+   '_cosine.c',
+   'amos_wrappers.c',
+   'cdf_wrappers.c',
+   'specfun_wrappers.c',
+   'sf_error.c',
+  ],
+  c_args : [numpy_nodepr_api, use_math_defines],
+  include_directories: [inc_np, '../_lib', '../_build_utils/src', 'cephes'],
+  dependencies : [py3_dep, npymath_lib, lapack],
+  link_with: [
+    amos_lib,
+    cdflib_lib,
+    cephes_lib,
+    mach_lib,
+    specfun_lib,
+  ],
+  install : true,
+  subdir : 'scipy/special'
+)
+
+py3.extension_module('_comb',
+  '_comb.pyx',
+  dependencies : py3_dep,
+  install : true,
+  subdir : 'scipy/special'
+)
+
+py3.extension_module('_test_round',
+  '_test_round.pyx',
+  include_directories: [inc_np, 'cephes', '../_lib', '../_build_utils/src'],
+  dependencies : [py3_dep, npymath_lib],
+  install : true,
+  subdir : 'scipy/special'
+)
+
+
+npz_files = [
+  ['tests/data/boost', 'boost.npz'],
+  ['tests/data/gsl', 'gsl.npz'],
+  ['tests/data/local', 'local.npz'],
+]
+
+# FIXME: installed generated files into a Python package seems to be a pain
+#        `install_data()` and `custom_target()` will install into
+#        `install_dir`, which is relative to the prefix. But this is different
```

Comment:
Note for myself - Look into `install_data`. Seems like it can be used to install `npz` files.